views/default/object/widget/elements/settings.php

```diff
@@ -16,7 +16,6 @@
 	'class' => [
 		$additional_class,
 	],
-	'prevent_double_submit' => true,
```

Comment:
In Elgg 4 the default behaviour will change to `true`, so could you please change this to explicit `false` to prevent the same error in the future.